### PR TITLE
Use block indent for cond statements

### PR DESCRIPTION
**Pull request by carylee**
After [this change](https://github.com/amperity/app/pull/4184) to our style guide, cljfmt and our style guide diverged in our style expectations for muli-line cond statements.

This change doesn't enable cljfmt to reformat multi-line conds with the correct spacing, but it does stop it from adding the no-longer-recommended hanging indents for cond expressions.

### CODE REVIEW

**Review by aengelberg — APPROVED** [review]

File: cljfmt/resources/cljfmt/indents/clojure.clj [hunk 1]
```diff
@@ -4,13 +4,13 @@
  as->            [[:block 1]]
  binding         [[:block 1]]
  bound-fn        [[:inner 0]]
- case            [[:cond 1]]
```
Comment (greglook) [inline cljfmt/resources/cljfmt/indents/clojure.clj:7]: I'd suggest we rename this rule, too - something like 'stair' maybe?

**Review by greglook — APPROVED** [review]
:+1:

File: README.md [hunk 2]
```diff
@@ -233,12 +233,12 @@ But indents at a constant two spaces otherwise:
 
 #### Cond rules
```
Comment (greglook) [inline README.md:234]: Update the header here too.

File: cljfmt/src/cljfmt/indent.cljc [hunk 3]
```diff
@@ -158,22 +158,22 @@
   [[rule-key _]]
   (cond
     (symbol? rule-key)
-      (if (namespace rule-key)
-        (str 0 rule-key)
-        (str 1 rule-key))
+    (if (namespace rule-key)
```
Comment (greglook) [inline cljfmt/src/cljfmt/indent.cljc:161]: Heh, meta.

File: cljfmt/src/cljfmt/indent.cljc [hunk 4]
```diff
@@ -197,13 +197,13 @@
         gp  (-> zloc z/up z/up)]
     (cond
       (zl/reader-conditional? gp)
-        (coll-indent zloc)
+      (coll-indent zloc)
       (#{:list :fn} tag)
```
Comment (greglook) [inline cljfmt/src/cljfmt/indent.cljc:201]: Maybe add blank lines here to help distinguish the forms?

File: cljfmt/src/cljfmt/indent.cljc [hunk 5]
```diff
@@ -283,7 +283,7 @@
 
 ;; ## Cond Style Rule
```
Comment (greglook) [inline cljfmt/src/cljfmt/indent.cljc:284]: Similar, update the comment header.

**Review by greglook — APPROVED** [review]

File: cljfmt/test/cljfmt/core_test.cljc [hunk 6]
```diff
@@ -24,15 +24,15 @@
   (testing "cond indentation"
     (is (= "(cond\n  a? a\n  b? b)"
            (reformat-string "(cond  \na? a\n   b? b)")))
-    (is (= "(cond\n  a?\n    a\n  b?\n    b)"
+    (is (= "(cond\n  a?\n  a\n  b?\n  b)"
```
Comment (greglook) [inline cljfmt/test/cljfmt/core_test.cljc:27]: With these test changes, you're no longer exercising the `:stair` rule - maybe fix them up or add a separate block of tests for stair-formatted `cond`?

**Comment by carylee** [comment]
I haven't had time to address the remaining comments and get the code coverage back up (though I've started). In order to get cljfmt back into a good state with respect to our style guide, I'm going to go ahead and merge this and address the remaining issues in a follow up PR.

**Pull request merged**